apps/internal/version/version.go

```diff
@@ -5,4 +5,4 @@
 package version
 
 // Version is the version of this client package that is communicated to the server.
-const Version = "0.5.0"
+const Version = "0.5.1"
```

Comment:
The code change is a patch, but we also raised the minimum required Go version
```suggestion
const Version = "0.6.0"
```